core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.html

```diff
@@ -59,633 +60,681 @@ <h1 class="stat-value"><[$ctrl.subscriptionsList.length]></h1>
+        <div *ngIf="activeSection === LEARNER_DASHBOARD_SECTION_I18N_IDS.INCOMPLETE && activeSubsection === LEARNER_DASHBOARD_SUBSECTION_I18N_IDS.COLLECTIONS">
           <span class="oppia-learner-dashboard-nonexistent-exploration"
-                ng-if="$ctrl.numberNonexistentIncompleteCollections !== 0">
-            <span translate="I18N_LEARNER_DASHBOARD_NONEXISTENT_INCOMPLETE_COLLECTIONS"
-                  translate-values="{numberNonexistent: $ctrl.numberNonexistentIncompleteCollections}"
-                  translate-interpolation="messageformat">
+                *ngIf="numberNonexistentIncompleteCollections !== 0">
+            <span [innerHTML]="'I18N_LEARNER_DASHBOARD_NONEXISTENT_INCOMPLETE_COLLECTIONS' | translate: {numberNonexistent: numberNonexistentIncompleteCollections}: messageformat">
             </span>
-            &#9785
+            &#9785;
           </span>
 
-          <md-card layout="row"
-                   class="oppia-learner-dashboard-no-activity-card"
-                   ng-if="$ctrl.incompleteCollectionsList.length === 0">
+          <mat-card class="oppia-learner-dashboard-no-activity-card layout-row"
+                   *ngIf="incompleteCollectionsList.length === 0">
             <div>
-              <p ng-if="$ctrl.noCollectionActivity" translate="I18N_LEARNER_DASHBOARD_NO_ACTIVITY_IN_COLLECTION"></p>
-              <p ng-if="!$ctrl.noCollectionActivity" translate="I18N_LEARNER_DASHBOARD_EMPTY_INCOMPLETE_COLLECTIONS"></p>
+              <p *ngIf="noCollectionActivity"
+                 [innerHTML]="'I18N_LEARNER_DASHBOARD_NO_ACTIVITY_IN_COLLECTION' | translate">
+              </p>
+              <p *ngIf="!noCollectionActivity"
+                 [innerHTML]="'I18N_LEARNER_DASHBOARD_EMPTY_INCOMPLETE_COLLECTIONS' | translate">
+              </p>
 
               <a class="btn oppia-dashboard-intro-button oppia-transition-200 oppia-learner-dashboard-intro-button-link"
-                 translate="I18N_ACTION_BROWSE_LESSONS"
-                 ng-href="/community-library">
+                 [innerHTML]="'I18N_ACTION_BROWSE_LESSONS' | translate"
+                 [href]="community-library">
               </a>
             </div>
-          </md-card>
+          </mat-card>
 
-          <div ng-repeat="tile in $ctrl.incompleteCollectionsList"
+          <div *ngFor="let tile of incompleteCollectionsList"
                class="oppia-exploration-summary-tile-container"
-               ng-mouseenter="showRemoveIcon=true" ng-mouseleave="showRemoveIcon=false">
+               (mouseenter)="tile.toggle = !tile.toggle "
+               (mouseleave)="tile.toggle = !tile.toggle">
             <i class="remove-icon fa fa-times"
-               ng-show="showRemoveIcon || $ctrl.checkMobileView()"
+               *ngIf="(tile.toggle || checkMobileView())"
```

Comment:
Do you need the brackets?

```suggestion
               *ngIf="tile.toggle || checkMobileView()"
```

Reply:
Nope, removed this and all others.